admin/app/components/solidus_admin/sidebar/account_nav/component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Account navigation
+class SolidusAdmin::Sidebar::AccountNav::Component < SolidusAdmin::BaseComponent
+  # @param user [#name]
+  # @param account_path [String]
+  # @param logout_path [String]
+  def initialize(user: Struct.new(:name).new("Alice Doe"), account_path: "#", logout_path: "#")
```

Comment:
This is very simple already and that's great 🤩 
You think this component could accept a string for the user's display name and require all arguments?

Reply:
Good call. As said above, I changed the param to a `String`. For now, I'd like to leave the params with their defaults defined. As that's dummy data, I'd prefer them to be scoped to the component. Once we have authentication, we can move stuff to the caller, although probably `account_path`, `logout_path` & `logout_method` will default to configuration options.